README.md

```diff
@@ -158,12 +163,19 @@ Pre-built docker images are available at https://hub.docker.com/r/fphammerle/swi
 
 Annotation of signed tags `docker/*` contains docker image digests: https://github.com/fphammerle/switchbot-mqtt/tags
 
+**Build**
 ```sh
 $ docker build -t switchbot-mqtt .
-$ docker run --name spelunca_switchbot \
-    --userns host --network host \
-    switchbot-mqtt:latest \
-    switchbot-mqtt --mqtt-host HOSTNAME_OR_IP_ADDRESS
+```
+
+**Install**
+```sh
+$ docker run -d \
+    --name switchbot \
+    --userns host \
+    --network host \
+    --restart=unless-stopped \
+    fphammerle/switchbot-mqtt --mqtt-host HOSTNAME_OR_IP_ADDRESS
```

Comment:
image name in `docker run` (`fphammerle/switchbot-mqtt`) does not match image name in `docker build` above (`switchbot-mqtt`)

Reply:
right, rename it to `switchbot-mqtt` and it should be fine